motoman_driver/MotoPlus/CtrlGroup.c

```diff
@@ -383,8 +442,8 @@ void Ros_CtrlGroup_ConvertToMotoPos(CtrlGroup* ctrlGroup, float radPos[MAX_PULSE
 	// Initilize memory space
 	memset(pulsePos, 0x00, sizeof(long)*MAX_PULSE_AXES);
 	
-	// Adjust joint order for 7 axis robot
-	if((ctrlGroup->groupId >= MP_R1_GID) && (ctrlGroup->groupId <= MP_R4_GID) && (ctrlGroup->numAxes == 7))
+	// Adjust joint order for 7 axis robot (SLEURBTE > SLURBTE)

```

Comment:
`strlen('SLEURBTE') == 8`, not `7`? I think there's an `E` too many in there somewhere.